smack-extensions/src/main/java/org/jivesoftware/smackx/softwareinfo/element/package-info.java

```diff
@@ -0,0 +1,17 @@
+/**
+ *
+ * Copyright 2019 Aditya Borikar
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jivesoftware.smackx.softwareinfo.element;
```

Comment:
There should be javadoc in the `package-info.java`. After all, that is its sole purpose.